### ISSUE TITLE
Description for start project doesn't make sense

### ISSUE THREAD

**Issue by derekflint** [deeplearninc/auger-ai#41]
Doesn't it just start the project? 

start() - starts Project cluster. DataSet processing, Experiment runs and Model deploy and predict need cluster to perform operations and will start cluster automatically. It is possible, but not necessary, to start cluster beforehand. Throws exception if can't validate credentials or network connection error.

**Comment by IgorFlint** [comment]
Yep, it just starts the project, so the command is optional for better control (if one wants to start/stop cluster by hands). Due to augerai.cli supposed to be more developer-oriented CLI. For end-user we have a2ml CLI.

Could you tell me what is wrong with it? And which description do you mean?

**Comment by vlad-dplrn** [comment]
@derekflint Please be more specific
